src/types/nft.ts

```diff
@@ -3,10 +3,20 @@ import { CryptoCurrencyIds } from ".";
 
 export type NFTStandards = "ERC721" | "ERC1155";
 
+export type NFTMediaSizes = "preview" | "big" | "original";
+
+export type NFTMedias = Record<
+  NFTMediaSizes,
+  {
+    uri: string;
+    mediaType: string; // mime-type
+  }
+>;
+
 export type NFTMetadata = {
   tokenName: string | null;
   nftName: string | null;
-  media: string | null;
+  medias: NFTMedias;
```

Comment:
I'm just concerned that impacts the persisted cache (what's in app.json on LLD).
For safety, migration should be tested from a build without this change, with some data in cache, to a build with this change.

Reply:
`accounts.nfts` is recreated from scratch each time, so nfts are redefined completely each sync. From all my testing it was never an issue, even with accounts already in cache 👌